Project/MSVC2022/Dll/MediaInfoDll.vcxproj

```diff
@@ -46,6 +46,7 @@
     <ProjectGuid>{BE6D3EF8-2F82-4F1E-956A-890C3614A2D5}</ProjectGuid>
     <RootNamespace>MediaInfoDll</RootNamespace>
     <Keyword>Win32Proj</Keyword>
+    <WindowsTargetPlatformVersion>10.0</WindowsTargetPlatformVersion>

```

Comment:
Seems useless as 10.0 is default, and is not future proof, I don't think it should be there (2x).